src/plugins/markdown.js

```diff
@@ -3,7 +3,7 @@ import hljs from "highlight.js";
 
 import "highlight.js/styles/atom-one-dark.css";
 
-marked.setOptions({
+marked.default.setOptions({
```

Comment:
This doesn't seem to relate to the update for the schema, and broke the build. This showed up in the console.

Not opposed to getting things that need to be fixed into another PR, but they need to be tested very carefully.

```
Uncaught TypeError: marked__WEBPACK_IMPORTED_MODULE_0___default.a.default is undefined
    <anonymous> markdown.js:6
    js app.js:1587
    __webpack_require__ app.js:854
    fn app.js:151
    <anonymous> index.js:1
    js app.js:1575
    __webpack_require__ app.js:854
    fn app.js:151
    <anonymous> main.ts:19
    ts app.js:1551
```

Reply:
sorry, I will check that issue, I notice there was an update in the markdown package